.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: 3p9grFir7YbLeurQqAqCuBgAtGosY2kc9
```

Comment:
Let's not use sensitive environmental variables on github. https://circleci.com/docs/1.0/code-coverage/#coveralls
https://docs.coveralls.io/
https://github.com/nickmerwin/node-coveralls